app/code/Magento/Checkout/Block/Cart/Totals.php

```diff
@@ -70,7 +88,7 @@ public function getJsLayout()
             $this->jsLayout = $processor->process($this->jsLayout);
         }
 
-        return json_encode($this->jsLayout, JSON_HEX_TAG);
+        return $this->serializer->serialize($this->jsLayout);
```

Comment:
Are you sure that `JSON_HEX_TAG` option can be omitted here?

Reply:
Hi @sivaschenko. As far as I understood the serializer should do the work of serializing standard json serialized data in Magento, shouldn't it?

In another PR I've been requested to change from json_encode/json_decode to serializer, so I thought it is being standardize like this.

Reply:
@tiagosampaio you are right, however, there are slight differences in serialization with JSON_HEX_TAG option. There is a pull request introducing the serializer with this option: https://github.com/magento/magento2/pull/16154 I think it might be better to wait unitl that pull request is merged and then use the JsonHexTag serializer for these changes.

Let me know what you think

Reply:
Ok @sivaschenko. After the merge I'll refactor the code to accomplish this. Thanks.

Reply:
@sivaschenko the fix is done. I've merged all the new 2.3-develop code base and applied the fix so that this class is using the proper serializer class.